pkg/runner/runner.go

```diff
@@ -92,6 +94,15 @@ func Run(cfg *Config) error {
 		grpcServer := grpc.NewServer()
 		otlpgrpc.RegisterTracesServer(grpcServer, api.NewTraceServer(client))
 
+		queryPlugin := shared.StorageGRPCPlugin{
+			Impl: jaegerquery.New(client.Connection),
+		}
+		err := queryPlugin.GRPCServer(nil, grpcServer)
+		if err != nil {
+			log.Error("msg", "Creating jaeger query GRPC server failed", "err", err)
+			return err
+		}
+
```

Comment:
we should log a similar message like we are doing for `line 107`

Reply:
this is not like 107 since 107 is inside a go func() { block

Reply:
No, the parent should log. That's what we do everywhere else.